File: app/src/main/java/io/moonshard/moonshard/helpers/ChatsHelper.java
package io.moonshard.moonshard.helpers;

import org.jivesoftware.smack.roster.RosterEntry;

import java.util.List;
import java.util.Set;

import io.moonshard.moonshard.MainApplication;
import io.moonshard.moonshard.models.roomEntities.ChatEntity;

 public  class ChatsHelper {

    public  List<ChatEntity> loadLocalChats() {
        return MainApplication.getChatDB().chatDao().getAllChats();
    }

    public  Set<RosterEntry> getRemoteContacts() {
        if(AppHelper.getXmppConnection() != null) {
            return AppHelper.getXmppConnection().getContactList();
        }
        return null;
    }

}

File: app/src/main/java/io/moonshard/moonshard/helpers/ChatsHelper.kt
package io.moonshard.moonshard.helpers

import androidx.lifecycle.LiveData
import io.moonshard.moonshard.MainApplication
import io.moonshard.moonshard.models.roomEntities.ChatEntity
import org.jivesoftware.smack.roster.RosterEntry

class ChatsHelper {
    fun loadLocalChats(): LiveData<List<ChatEntity>> {
        return MainApplication.getChatDB().chatDao().getAllChats()
    }

    val remoteContacts: Set<RosterEntry>?
        get() = if (MainApplication.getXmppConnection() != null) {
            MainApplication.getXmppConnection().contactList
        } else null
}